File: 1-easy/problems/0976-largest-perimeter-triangle/solution.ts
function largestPerimeter(A: any): boolean | number | string | any {
  """
  :type A: List[int]
  :rtype: int
  """
  A.sort()
  for (i in reversed(Array.from({length: (A}, (_, i) => i).length - 2))) {
      if (A[i] + A[i+1] > A[i+2]) {
          return A[i] + A[i+1] + A[i+2];
  return 0;
}

export { largestPerimeter };

File: 1-easy/problems/0976-largest-perimeter-triangle/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def largestPerimeter(self, A):
        """
        :type A: List[int]
        :rtype: int
        """
        A.sort()
        for i in reversed(range(len(A) - 2)):
            if A[i] + A[i+1] > A[i+2]:
                return A[i] + A[i+1] + A[i+2]
        return 0
